gpfit/xfoil_wrapper.py

```diff
@@ -0,0 +1,124 @@
+" python wrapper to call xfoil "
```

Comment:
nice! @mayork, @1ozturkbe, or @cjk12, could y'all check if this particular xfoil-calling function is something you'd find useful?